### DISCUSSION TITLE
Unicode in JSX string props are being escaped

### DISCUSSION THREAD

**Discussion by GCandez** [parcel-bundler/parcel#6687] [Q&A] [answered]
If I have a component like this:
```JSX
const SomeComponent = ({value}) => <h1>{value}</h1>
<SomeComponent value="ação" />
````
The "ação" string gets escaped to `a\xe7\xe3o`

However, if I use children...
```JSX
const SomeComponent = ({children}) => <h1>{children}</h1>
<SomeComponent>ação</SomeComponent>
````

It works as expected.
This issue cropped up when I updated to `2.0.0-rc.0`, it works normally on `2.0.0-beta.3`

Wanted to ask here first before creating an issue because it may be some config I haven't set properly... I searched the docs and issues but couldn't find anything...

**Comment by mischnic** [comment, marked as answer]
This should work correctly without any special settings. We might have to bump our swc version
